demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2205,6 +2216,16 @@ export class DemoMeetingApp
       }
     });
 
+    // If this is a content share stream, it might be our own, from audio that
+    // is already playing locally (e.g., in a tab or app).
+    // In this case, we need to mute it, otherwise we'll hear our own shared audio.
+    // If it's not, we unmute the element — it should always be silent anyway, but
+    // this is a consistent default.
+    const us = this.meetingSession.configuration.credentials.attendeeId;
+    const isOurContent = tileState.isContent && tileState.boundAttendeeId === `${us}#content`;
+    this.log(`Is our content, and thus should be muted? ${isOurContent}`);
+    videoElement.muted = isOurContent;
+
     this.log(`binding video tile ${tileState.tileId} to ${videoElement.id}`);
     this.audioVideo.bindVideoElement(tileState.tileId, videoElement);
```

Comment:
I would expect `bindVideoElement` to mute the video element https://github.com/aws/amazon-chime-sdk-js/blob/master/src/videotile/DefaultVideoTile.ts#L43 ?

Reply:
That's what I thought, but it's missing a bunch of the information it needs to make that determination, so I would need to alter the tile info structure to get to that, right?

Reply:
I do not follow. Like Xuan said, bindVideoElement should just call connectVideoStreamToVideoElement and check whether the videoElement is muted and if it is not, append the 'muted' attribute. Is that not the case?

Reply:
Unless we want to always mute every video element (which seems to perhaps be the case!) we don't know within that code whether this is _our_ content stream, and thus we should mute it.

There is already code in `DefaultVideoTile` in `connectVideoStreamToVideoElement` to set `muted` by default, so perhaps that's not being called?